### PR TITLE
Updated PowerBI Report and Setup Documentation

### PR DESCRIPTION
**Pull request by rguo123**
This PR contains an updated PowerBI Template file with the calculated columns and example visualizations included. Setup information has also been added to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,23 @@
...
+
+## Setup Instructions
+
+1. Download [PowerBI Desktop](https://www.microsoft.com/en-us/download/details.aspx?id=58494). Note: a Windows machine or VM will be needed to run the application.
+
+2. Open `AzureDevOpsPipelines-Reporting-PowerBI.pbit`.
+
+3. Once the file is opened, you will be prompted to provide an AzDO Organization name. The pipelines for all readable projects under this organization will be used for the report.
+
+4. After providing an organization, the app will prompt you to sign in to Azure DevOps. For more information, see [here](https://docs.microsoft.com/en-us/azure/devops/report/powerbi/access-analytics-power-bi?view=azure-devops). Note: the user access you sign in with determines which projects and pipelines will be loaded from AzDO.
```
Comment (NarmathaBala) [inline README.md]: The PAT configuration is good to mention. For a business application use, it is better to provide an alternate than user credentials.

**Review by NarmathaBala — CHANGES_REQUESTED** [review]
Other than the PAT comment, everything is looking good. Can be merged after adding PAT related note.

Thanks for your contribution to this PR.

File: README.md (hunk 1 above)
Comment (rguo123) [reply README.md]: I updated the README to mention our recommendation is to use PATs. I linked a doc that points out more on the security reasons and also how to create and configure a PAT. Let me know if you think I should specify more information. Thanks!

**Review by NarmathaBala — APPROVED** [review]
LGTM. Thanks again for your contribution

**Pull request merged**